components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -2525,6 +2525,15 @@ interface BraveWalletService {
   GetNetworkForSelectedAccountOnActiveOrigin() => (NetworkInfo? network);
   SetNetworkForSelectedAccountOnActiveOrigin(string chain_id) => (bool success);
 
+  // At this time only coin will be used to manage chain id setting,
+  // so any account can be used.
+  // But in the future we are going to make unique relation
+  // between origin, account and chain.
+  GetNetworkForAccountOnActiveOrigin(AccountId account)
+    => (NetworkInfo? network);
+  SetNetworkForAccountOnActiveOrigin(AccountId account, string chain_id)
+    => (bool success);
+
   // Filters accounts with permissions on active origin.
```

Comment:
I believe we are going to deprecate  
`GetNetworkForSelectedAccountOnActiveOrigin() => (NetworkInfo? network);
  SetNetworkForSelectedAccountOnActiveOrigin(string chain_id) => (bool success);`
in the future? maybe we want to add some comment regarding to that as well

Reply:
In the mojo interface or here?

Reply:
here is fine.